packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -615,19 +615,9 @@ async function fluid({ file, args = {}, reporter, cache }) {
   }
 
   // when using maxHeight and/or maxWidth determine appropriate aspect ratios
-  let maxWidth
-  let maxHeight
   if (!options.aspectRatio) {
     options.aspectRatio = presentationWidth / presentationHeight
   }
-  if (options.maxHeight || options.maxWidth) {
```

Comment:
@wardpeet looking at `gatsby-plugin-sharp` I'm not seeing maxHeight/maxWidth being returned anymore for fluid image types, or the dimensions being calculated if only one (maxHeight or maxWidth) is used.  So maybe I'm just lost at this point on the new changes :)